### PR TITLE
[Filebeat] Changes to text fields in elasticsearch module

### PR DESCRIPTION
**Pull request by ycombinator**
This PR is an offshoot of conversations and decisions made in https://github.com/elastic/beats/pull/10372 w.r.t `text` fields, but scoped to the `elasticsearch` module.

### CODE REVIEW

**Comment by ycombinator** [comment]
jenkins, test this

File: filebeat/module/elasticsearch/slowlog/ingest/pipeline.json [hunk 1]
```diff
@@ -19,6 +19,14 @@
               ]
           }
       },
+      {
+          "split": {
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/ingest/pipeline.json]: Nice

File: filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log-expected.json [hunk 2]
```diff
@@ -9,7 +9,6 @@
         "elasticsearch.slowlog.logger": "index.indexing.slowlog.index",
         "elasticsearch.slowlog.routing": "",
         "elasticsearch.slowlog.source_query": "{\"@timestamp\":\"2018-07-04T21:50:40.799Z\",\"metricset\":{\"module\":\"system\",\"rtt\":9610,\"name\":\"network\"},\"system\":{\"network\":{\"name\":\"bridg\",\"in\":{\"packets\":0,\"errors\":0,\"dropped\":0,\"bytes\":0},\"out\":{\"errors\":0,\"dropped\":0,\"packets\":1,\"bytes\":342}}},\"beat\":{\"name\":\"Rados-MacBook-Pro.local\",\"hostname\":\"Rados-MacBook-Pro.local\",\"version\":\"6.3.0\"},\"host\":{\"name\":\"Rados-MacBook-Pro.local\"}}",
-        "elasticsearch.slowlog.took": "221micros",
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log-expected.json]: One thing I spot now: Below the message fields seems to contain everything including timestamp. Is that expected?

For the `took`. The plan is to extract the unit later or just remove it?

Comment (ycombinator) [reply filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log-expected.json]: I didn't follow your question about the `message` field?

For the `took`, I think we should just the unit as we already have the actual numeric value in `ms`, which seems more useful?

Comment (webmat) [reply filebeat/module/elasticsearch/slowlog/test/auditlog_index_indexing_slowlog.log-expected.json]: The point about message is that once the header has been parsed and moved to other fields, only the rest of the message should be left in `message`.

Right now, the `message` fields looks like an `event.original`, which is meant to contain the pristine initial copy of the event (if the user wants this).

For the `took` field, I would keep it until we're ready to interpret it, no? Right now event.duration is based on the took_millis (removed by the pipeline), for simplicity, which is often rounded to 0ms. Elasticsearch is too fast. But I think it's worth keeping the custom field with the more precise timing until we can leverage it better (or until the ES log format gives us nanoseconds).

**Comment by ycombinator** [comment]
jenkins, test this

**Pull request merged**